tests/foreman/api/test_classparameters.py

```diff
@@ -184,13 +324,26 @@ def test_negative_update_parameter_type(self):
 
         1. Parameter not updated with string type for invalid value.
         2. Error raised for invalid default value.
-
-        @caseautomation: notautomated
         """
+        sc_param = self.sc_params_list.pop()
+        for test_data in invalid_sc_parameters_data():
+            with self.subTest(test_data):
+                with self.assertRaises(HTTPError) as context:
```

Comment:
@svtkachenko Just read what you said. Checked the [code](https://github.com/svtkachenko/robottelo/blob/312b8318c79df584ec7dfd415d0107906eb2aa80/tests/foreman/api/test_classparameters.py#L337)  and 2 assertions are in fact on assertRaises scope. So you need to move them to subtest scope if that is what you want.